src/products/components/ProductListPage/filters.ts

```diff
@@ -150,7 +162,22 @@ export function createFilterStructure(
       ),
       active: opts.productType.active
     },
-    ...opts.attributes.map(attr => ({
+    ...booleanAttributes.map(attr => ({
+      ...createBooleanField(
+        attr.slug as any,
```

Comment:
`as any` is only needed here because the output is typed incorrectly, it expects enum keys of `ProductFilterKeys` but a filter key can be any string (because they are applied dynamically). I'd change the function output type to be string instead and get rid of `as any`